tests/dlt_core_unit_tests.py

```diff
@@ -15,14 +16,25 @@
 class TestCoreStructures(unittest.TestCase):
 
     def setUp(self):
-        self.size_map = {'cDltServiceConnectionInfo': 10,
-                         'cDltStorageHeader': 16,
-                         'cDltStandardHeader': 4,
-                         'cDltStandardHeaderExtra': 12,
-                         'cDltExtendedHeader': 10,
-                         'cDLTMessage': 120,
-                         'cDltReceiver': 64,
-                         'cDltClient': 128}
+        api_ver = tuple(int(num) for num in API_VER_STR.split('.'))
```

Comment:
It could be a map.

Please put a map in the test class. Like

```
class TestCoreStruecutres(unites.TestCase)
    CORE_VER_SIZE_MAP = {
        (2, 18, 5): { ... },
        (2, 18, 6): { ... }, 
    }
```